src/config/app.ts

```diff
@@ -22,9 +22,13 @@ export const MAX_BYTES_FOR_MEMO = 639 // BOLT
 
 export const SATS_PER_BTC = 10 ** 8
 export const SAT_USDCENT_PRICE = "SAT-USDCENT-PRICE"
+export const USER_PRICE_UPDATE_EVENT = "USER-PRICE-UPDATE-EVENT"
+
 export const lnPaymentStatusEvent = (paymentHash: PaymentHash) =>
   `LN-PAYMENT-STATUS-${paymentHash}`
 
+export const walletUpdateEvent = (walletId: WalletId) => `WALLET_UPDATE-${walletId}`
```

Comment:
should we use this too for ln payment/receipt too? maybe this should contain balance and lnPaymentStatusEvent only the secret and invoice details

Reply:
It is used in ln updates too